release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -50,6 +50,11 @@ This release adds improvements related to the following components and concepts.
 
 The `coreos-installer` utility now has `iso customize` and `pxe customize` subcommands for more flexible customization when installing {op-system} on bare metal from the live ISO and PXE images.
 
+[id="ocp-4-10-coreos-installer-customize-ignition-ca"]
+==== {op-system} PXE installation can generate a custom `initramfs` image
+
+{op-system} PXE installation can now be configured to fetch from HTTPS servers that use a custom certificate authority or self-signed certificate. This can be done by generating a customized `initramfs` image by running `coreos-installer pxe customize --ignition-ca`.
+
```

Comment:
I would suggest incorporating this callout into the previous section.  For example:

```suggestion
The `coreos-installer` utility now has `iso customize` and `pxe customize` subcommands for more flexible customization when installing {op-system} on bare metal from the live ISO and PXE images. This includes customizing the installation to fetch Ignition configs from HTTPS servers that use a custom certificate authority or self-signed certificate.
```

Reply:
I'd be inclined to split the difference, combining it into the previous section but giving it a separate paragraph.  ("In particular, ..." or something similar.)  I think we want to emphasize it a little more than Micah's combined paragraph does, since this is a feature we've gotten a lot of requests for.